contracts/staking/BaseShareField.sol

```diff
@@ -253,9 +272,8 @@ contract BaseShareField is DSMath {
 		UserInfo storage userInfo = users[user];
 		_canMintRewards();
 		_update(rewardsPerBlock, blockStart, blockEnd);
-		_audit(user, userInfo.amount);
+		_audit(user, userInfo.amount, 1);
```

Comment:
1?

Reply:
Doesn't matter just needs to be something different than 0

Reply:
yes  i know, i can read the code.
what do we do when we write something that isnt obvious??

Reply:
Added comment